pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go

```diff
@@ -287,6 +293,16 @@ func (r *ReconcileNetworkAddonsConfig) renderObjects(networkAddonsConfig *opv1al
 	}
 	objs = append([]*unstructured.Unstructured{applied}, objs...)
 
+	// Label objects with version of the operator they were created by
+	for _, obj := range objs {
+		labels := obj.GetLabels()
+		if labels == nil {
+			labels = map[string]string{}
+		}
+		labels["networkaddonsoperator.network.kubevirt.io/version"] = operatorVersion
```

Comment:
You can take the prefix of the string from here https://github.com/kubevirt/cluster-network-addons-operator/blob/bdc8d36b05e55c084fb19fd0ed96e8359b960ab3/pkg/apis/networkaddonsoperator/v1alpha1/register.go#L13

Reply:
Done